lib/mongo/operation/specifiable.rb

```diff
@@ -131,6 +131,11 @@ module Specifiable
       # @since 2.0.0
       READ = :read.freeze
 
+      # Whether to bypass document level validation.
+      #
+      # @since 3.2.0 (?)
```

Comment:
this would be 2.2.0

https://github.com/mongodb/mongo-ruby-driver/blob/master/lib/mongo/version.rb#L20